falco/README.md

```diff
@@ -282,3 +330,26 @@ helm install falco \
   --set-file certs.ca.crt=/path/to/ca.crt \
   falcosecurity/falco
 ```
+
+## Generating SSL certificates for webserver (optional) or gRPC:
```

Comment:
It's required for grpc, can we massage this line so it is obvious that it's needed for grpc?

Reply:
It's required for both, but they are different setups.
gRPC can be configured with mutual TLS; on the other hand, you can basically enable HTTPS in the webserver.

I believe the better thing to do here is just to remove this block and add pointers to the official documentation:
- for gRPC :point_right: https://falco.org/docs/grpc/#generate-valid-ca
- for the webserver :point_right: https://github.com/civetweb/civetweb/blob/master/docs/OpenSSL.md#creating-a-self-signed-certificate

Those links are already provided by the "configuration" section within the doc (see https://falco.org/docs/configuration/).
@antoinedeschenes @nibalizer WDYT?

Reply:
PS
gRPC via unix socket does not require mTLS.

Reply:
The gRPC instructions seem to fit both use-cases (except the client cert, but that would be a good thing to add to falco), I used subjectAltName with an IP address in my example, but "CN=localhost" makes more sense.